apps/communications/dialer/test/marionette/keypad_test.js

```diff
@@ -40,19 +40,22 @@ marionette('Dialer > Keypad', function() {
     keypadTap(three);
   }
 
-  function keypadTap(elem, longPress) {
-    var number = subject.client.findElement(selectors.phoneNumber);
-    var length = number.getAttribute('value').length;
+  function typeLongNumber() {
+    typeNumber();
+    typeNumber();
+    typeNumber();
+    typeNumber();
+    typeNumber();
+  }
 
+  function keypadTap(elem, longPress) {
     if (longPress) {
       actions.longPress(elem, 1).perform();
     } else {
       actions.tap(elem).perform();
     }
 
-    client.waitFor(function() {
-      return (number.getAttribute('value').length == (length + 1));
-    });
+    subject.client.helper.wait(200);
```

Comment:
Because the length of the input not always change. It does not change after ellipsis acting. So I'm going to change this to wait for current value to be different to the former one if you don't mind.